test/test-lua.sh

```diff
@@ -25,14 +25,23 @@ set -o pipefail
 
 BUSTED_ARGS="-v --pattern=_test"
 
-resty \
-  -I ./rootfs/etc/nginx/lua \
-  --shdict "configuration_data 5M" \
-  --shdict "certificate_data 16M" \
-  --shdict "certificate_servers 1M" \
-  --shdict "ocsp_response_cache 1M" \
-  --shdict "balancer_ewma 1M" \
-  --shdict "balancer_ewma_last_touched_at 1M" \
-  --shdict "balancer_ewma_locks 512k" \
-  --shdict "global_throttle_cache 5M" \
-  ./rootfs/etc/nginx/lua/test/run.lua ${BUSTED_ARGS} ./rootfs/etc/nginx/lua/test/ ./rootfs/etc/nginx/lua/plugins/**/test
+SHDICT_ARGS=(
+    "-I" "./rootfs/etc/nginx/lua"
+    "--shdict" "configuration_data 5M"
+    "--shdict" "certificate_data 16M"
+    "--shdict" "certificate_servers 1M"
+    "--shdict" "ocsp_response_cache 1M"
+    "--shdict" "balancer_ewma 1M"
+    "--shdict" "quota_tracker 1M"
+    "--shdict" "high_throughput_tracker 1M"
+    "--shdict" "balancer_ewma_last_touched_at 1M"
+    "--shdict" "balancer_ewma_locks 512k"
+    "--shdict" "global_throttle_cache 5M"
+    "./rootfs/etc/nginx/lua/test/run.lua"
+)
+
+if [ $# -eq 0 ]; then
```

Comment:
Fun fact: I always struggle with bash so chatgpt wrote this for me:

<img width="731" alt="Screenshot 2023-01-24 at 11 35 45 AM" src="https://user-images.githubusercontent.com/522155/214235812-d9a97384-f36d-402d-9e25-92aa6f5f1ec1.png">

Reply:
As well as the array logic:

<img width="777" alt="Screenshot 2023-01-24 at 11 36 22 AM" src="https://user-images.githubusercontent.com/522155/214235855-0064815c-e59f-46ef-be98-e1c9496fcd49.png">